src/tic_tac_toe/board.clj

```diff
@@ -2,10 +2,30 @@
 
```

Comment:
What is the single responsibility of the board namespace? Do you adhere to SRP as it stands?